# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/WorkerPrivate.cpp
# Commit: 4f2ec6bc75c1
# Full Hash: 4f2ec6bc75c142b1633a8ae2a98b70fd5215ed1f
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2022-02-27 09:40:54
# Regressor Bug: 1754365
# File Overlap Count: 1
# Description:
#   Bug 1754365 - Saving NavigationPreload's PerformanceTimingData into worker's PerformanceStorage. r=dom-worker-reviewers,jesup
#   
#   Depends on D138251
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138252
# ==============================================================================

diff -r 8fec60d0c499 -r 4f2ec6bc75c1 dom/workers/WorkerPrivate.cpp
--- a/dom/workers/WorkerPrivate.cpp	Sun Feb 27 00:26:22 2022 +0000
+++ b/dom/workers/WorkerPrivate.cpp	Sun Feb 27 00:26:22 2022 +0000
@@ -5375,7 +5375,6 @@
 }
 
 PerformanceStorage* WorkerPrivate::GetPerformanceStorage() {
-  AssertIsOnMainThread();
   MOZ_ASSERT(mPerformanceStorage);
   return mPerformanceStorage;
 }